### DISCUSSION TITLE
Deck Expansion

### DISCUSSION THREAD

**Discussion by MINJEEgi** [bitcraze/discussions#1201] [Q&A] [answered]
Hello, I have a question and wonder if it's possible to fly with three decks connected? I want to use the AI deck, Flow deck, and MultiRanger deck all together for a project. I've seen that expansion up to two decks is possible, but I'm curious if three can also be connected. I would really appreciate an answer.

**Comment by knmcguire** [comment]
Hi! As long as the pins don't collide it is possible! Check out this page for that: https://www.bitcraze.io/documentation/system/platform/cf2-expansiondecks/#deck---deck

Mind that there is going to be an update on that table soon so also look at the source of it: https://github.com/bitcraze/bitcraze-website/blob/master/src/documentation/system/platform/cf2-expansiondecks.md#deck---deck

From that, it seems that that aideck, multiranger and flowdeck should work but you'll need to have pins large enough (like the [ extra long pins](https://store.bitcraze.io/collections/spare-parts/products/male-long-deck-connector)), and I hope that the crazyflie is able to carry it. Perhaps with the thrust upgrade kit? https://store.bitcraze.io/collections/bundles/products/thrust-upgrade-bundle-for-crazyflie-2-x

**Reply by MINJEEgi** [reply]
Yes i'm gonna use thrust upgrade kit! And I have one more question. I noticed the 350mAh battery, should i use it if I want to operate three decks? It seems to be out of stock, so when can I purchase those things?

**Reply by ataffanel** [reply]
Yes, the bigger battery would help but we are unfortunately currently out of stock and it will take a while to come back. In the mean time you can have a look at this blog post for alternative batteries that would also work: https://www.bitcraze.io/2023/06/new-350ma-li-po-on-the-way/

Your Crazyflie should fly with the old battery, it will just limit the fly time.

**Reply by MINJEEgi** [reply]
Thank you for your reply. Is there a way to know when the stock will be replenished? I was also able to confirm that the alternative battery you sent is out of stock. And is it possible to use long deck connector with battery lipo that I already have? I know it will limit the fly time.

https://store.bitcraze.io/collections/spare-parts/products/male-long-deck-connector https://store.bitcraze.io/collections/spare-parts/products/250mah-lipo-battery

**Reply by knmcguire** [reply, marked as answer]
Unfortunately the 350 battery is expected back in stock early may. See the announcement in the latest blogpost https://www.bitcraze.io/2024/03/lambdaflight-crazyflie-meets-haskell/
